coral/src/app/features/approvals/acls/AclApprovals.tsx

```diff
@@ -123,6 +123,7 @@ function AclApprovals() {
         topic,
       }),
     onSuccess: (data) => {
+      queryClient.refetchQueries(["getRequestsWaitingForApproval"]);
```

Comment:
Not for now, but what do you think about having these query keys as consts somewhere? It feels weird just being able to add any string here.  Or have the queries in their own file  (why did we even abandon this 🤔 ) Or go custom hook all the way again and query keys are located with the hooks ^^

Reply:
Yes maybe we went a little too far in the "all queries inline" thing. 

Maybe a `react-queries` file in `domain could do something like
```ts
const queries = {
  approveTopicRequest: {
    queryFn: approveTopicRequest,
    baseQueryKeys: ["approveTopicRequest"],
    refetch: queryClient.refetchQueries(["approveTopicRequest"]),
  },
};

// usage

useQuery({
  queryFn: queries.approveTopicRequest.queryFn,
  queryKey: [
    ...queries.approveTopicRequest.baseQueryKeys,
    "any",
    "other",
    "keys",
  ],
});

/// ...
queries.approveTopicRequest.refetch()
```

this is kind of similar to what we had before, but maybe the reponsibility is a bit more strictly defined?

Reply:
Something like that! Let's sync about that after the release (and after my vacation :D ) in the frontend team. It's not important for this PR right now, just noticed it here again & wanted to bring it up.